### PR TITLE
Wbps18.pre processing

### PR DESCRIPTION
**Pull request by digrigor**

### CODE REVIEW

**Comment by digrigor** [comment]
Awaiting review by @mvvucak

File: parasite/modules/python/ProductionUtils.py [hunk 1]
```diff
@@ -136,4 +137,17 @@ def url_file_exists(path):
 def findOccurrences(s, ch):
     return [i for i, letter in enumerate(s) if letter == ch]
 
+def download_if_url_or_copy_file_command(path, output=None):
+    if validators.url(path):
+        command = "curl -H 'Connection: keep-alive' --keepalive-time 2 " + \
```
Comment (mvvucak) [inline parasite/modules/python/ProductionUtils.py:142]: Only suggestion here is to use str.format() for readability. Minor.

File: parasite/scripts/production/core-creation/gff_utils.py [hunk 2]
```diff
@@ -22,15 +22,13 @@
...
 # Removes provided prefixes from all columns in a .gff3 dataframe.
 def remove_prefixes_from_column_values(gff_df, prefixes):
-    for prefix in prefixes:
-        # Look only at beginning of line, escape any special characters.
-        prefix = "^"+re.escape(prefix)
-        for col in gff_df:
-            # Avoid using re.sub on NaN values as this turns them into str
-            nan_mask = gff_df.loc[:,col].isna()
-            gff_df.loc[~nan_mask, col] = [re.sub(prefix, "", str(x)) for x in gff_df.loc[~nan_mask, col]]
+    gff_df = gff_df.replace(prefixes, "", regex=True)
```
Comment (mvvucak) [inline parasite/scripts/production/core-creation/gff_utils.py:25]: Does this still check that the prefixes being replaced are at the beginning of the value?

File: parasite/scripts/production/rnaseq/get_ena_studies_by_taxon_id.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import requests
+from ProductionUtils import *
+from dependencies import *
+import time
+
+from optparse import OptionParser
```
Comment (mvvucak) [inline parasite/scripts/production/rnaseq/get_ena_studies_by_taxon_id.py:6]: May want to change this to argparse.
See: https://stackoverflow.com/questions/3217673/why-use-argparse-rather-than-optparse

**Pull request merged**